tests/test_conversion.py

```diff
@@ -313,6 +313,14 @@ def test_field_for(self, models, session):
         field = field_for(models.Student, "full_name", field_class=fields.Date)
         assert type(field) == fields.Date
 
+    def test_related_initialization_warning(self, models, session):
+        with pytest.warns(DeprecationWarning) as record:
+            Related(column=[])
+            assert (
+                record[0].message.args[0]
+                == "`column` parameter is deprecated and will be removed in future releases. Use `columns` instead."
+            )
+
```

Comment:
You could use `match` (https://pytest.org/en/6.2.x/warnings.html).

```py
with pytest.matches(DeprecationWarning, match="column parameter is deprecated...")
```

Do you think you can also add a test using `columns` ?

Reply:
sure will update

Reply:
updated with mentioned changes are these ok?